morf-core/src/main/java/org/alfasoftware/morf/jdbc/ConnectionResourcesBean.java

```diff
@@ -87,6 +89,8 @@ public class ConnectionResourcesBean extends AbstractConnectionResources {
   private String schemaName;
   private String userName;
   private int statementPoolingMaxStatements;
+  private int fetchSizeForBulkSelects;

```

Comment:
Should this be Integer instead of int? Some dialects might have a valid fetch size of 0